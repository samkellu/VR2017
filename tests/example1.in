./binary_files/01.bin 0xFF 0xAA 0x11
