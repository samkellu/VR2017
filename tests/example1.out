Delimiter byte 0 is: 255
Delimiter byte 1 is: 170
Delimiter byte 2 is: 17
Checksum is: 68

Chunk: 0 at offset: 0
    Packet: 0
        Data before swizzle -> B0: 123, B1: 222, B2: 113
        Swizzle: XZY
        Data after swizzle -> X: 123, Y: 113, Z: 222
    Chunk Average X: 123.00, Average Y: 113.00, Average Z: 222.00

